doc/index.rst

```diff
@@ -185,6 +185,9 @@ Note that some of these currently tied only to a single option, but we still lea
 * **CATKIN_PARALLEL_JOBS** (default: -p4): Maximum number of packages to be built in parallel that is passed to underlining build tool. As of Jan 2016, this is only enabled with `catkin_tools`. See for more detail about `number of build jobs <http://catkin-tools.readthedocs.org/en/latest/verbs/catkin_build.html#controlling-the-number-of-build-jobs>`_ and `documentation of catkin_tools <https://catkin-tools.readthedocs.org/en/latest/verbs/catkin_build.html#full-command-line-interface>`_ that this env variable is passed to internally in `catkin-tools`.
 * **CATKIN_PARALLEL_TEST_JOBS** (default: -p4): Maximum number of packages which could be examined in parallel during the test run. If not set it's filled by `ROS_PARALLEL_JOBS`.
 * **CCACHE_DIR** (default: not set): If set, `ccache <https://en.wikipedia.org/wiki/Ccache>`_ gets enabled for your build to speed up the subsequent builds in the same job if anything. See `detail. <https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#cache-build-artifacts-to-speed-up-the-subsequent-builds-if-any>`_
+* **COMMIT_IMAGE** (default: false): If set to true, the docker images which contains the build and test will be saved in the outer-layer docker which runs the script.  The commited image is then be available to be pushed to a docker registry.  This behavior enables a building Docker images with CI/CD approach. If set to false, or unset the container will not be commited and is removed.
+* **COMMIT_IMAGE_NAME** (default: not set): used to specify an image name during the docker commit command which is triggered by setting COMMIT_IMAGE=true. If unset and if COMMIT_IMAGE=true then container id will be used as the image name.
+* **COMMIT_IMAGE_MSG** (default: not set): used to specify a commit during the docker commit command which is triggered by setting COMMIT_IMAGE=true. If unset and if COMMIT_IMAGE=true then the commit message will be empty.
```

Comment:
> If unset and if COMMIT_IMAGE=true then the commit message will be empty.

Not sure if this is true. In [this line](https://github.com/ros-industrial/industrial_ci/pull/319/files#diff-67ae1351f65f3d5321aa8cc3782c7f2eR55) the variable seems to be filled with certain value..